app/views/days/index.html.erb

```diff
@@ -11,26 +11,51 @@
     __  ___                ________                __            
    /  |/  /___  ____  ____/ /_  __/________ ______/ /_____  _____
   / /|_/ / __ \/ __ \/ __  / / / / ___/ __ `/ ___/ //_/ _ \/ ___/
- / /  / / /_/ / /_/ / /_/ / / / / /  / /_/ / /__/ ,< /  __/ /    
+ / /  / / /_/ / /_/ / /_/ / / / / / /  / /_/ / /__/ ,< /  __/ /    
 /_/  /_/\____/\____/\__,_/ /_/ /_/   \__,_/\___/_/|_|\___/_/     
                                                                  
 </pre>
 
 
 <button type='button' class= "backbutton" onclick="location.href='<%= new_day_path() %>';">Create new daily entry</button> 
 
-<div id="piechartbutton" class="leftshift1"><button onclick="switchpiechart()">See pie chart</button></div>
+<div id="piechartbutton" class="leftshift1" style="margin-bottom: 1%;"><button onclick="switchpiechart()">See pie chart</button></div>
+<div id="linechartbutton" class="hidden leftshift1" style="margin-bottom: 1%;"><button onclick="switchlinechart()">See line chart</button></div>
+<div id="columnchartbutton" class="hidden leftshift1" style="margin-bottom: 1%;"><button onclick="switchcolumnchart()">See column chart</button></div>
+
+<div id="linechartbuttons">
+  <div style="display: flex; margin-left: 2%;" >
+    <div class="leftshift1"><button id="linechartbuttonA" class="buttonselect" onclick="switchlinechartall()">All</button></div>
+    <div class="leftshift1"><button id="linechartbuttonW" onclick="switchweek()">This week</button></div>
+    <div class="leftshift1"><button id="linechartbuttonM" onclick="switchmonth()">This month</button></div>
+    <div class="leftshift1"><button id="linechartbuttonY" onclick="switchyear()">This year</button></div>
+  </div>
+</div>
 
-<div id="linechartbutton" class="hidden leftshift1"><button onclick="switchlinechart()">See line chart</button></div>
+  <div id="linechart">
+  <%= line_chart Day.group_by_day(:mooddate, default_value: "missing").sum(:moodrating), library: {spanGaps: true}, min: 0, max: 10 %>
+  </div>
 
+    <div id="linechartweek" class="hidden">
```

Comment:
Maybe put these linechart by week, month, etc.. in partials but it works as is for now